test/unit/trigger-method.spec.js

```diff
@@ -141,8 +141,8 @@ describe('trigger event and method name', function() {
       this.childView.$el.click();
     });
 
-    it('should fire the event method once', function() {
-      expect(this.onChildviewFooClickStub).to.have.been.calledOnce;
+    it('should fire the event method', function() {
+      expect(this.onChildviewFooClickStub).to.have.been.called;
```

Comment:
I think we should prove this isn't an issue.  log a trace or something.. I do think there are child event issues being introduced by combining itemview and layoutview, but I am surprised we hadn't seen a failing test.